documentation/releases/2.0.0-alpha10.md

```diff
@@ -156,6 +156,15 @@ class YourNode(
         }
 ```
 
+## Fixed typo `canHandeBackPress`
+
+```diff
+interface AppyxComponent /*...*/ {
+    /*...*/
+-    fun canHandeBackPress(): StateFlow<Boolean>
++    fun canHandleBackPress(): StateFlow<Boolean>
+```
+
```

Comment:
We already released alpha10, this should go to a similar document but for alpha11